### ISSUE TITLE
votca/share folder has no csg subfolder

### ISSUE THREAD

**Issue by JensWehner** [votca/csg#350]
votca/share/votca has the following content 

```
csg-tutorials
ctp
kmc
moo
rc
scripts
template
xml
xtp
```

it should be

```
csg-tutorials
ctp
kmc
moo
csg---
       |
       rc
       scripts
       template
       xml
xtp

```

or not?

**Comment by junghans** [comment]
That is mostly historic, some of the scripts, e.g. `csg_call` are used by other modules as well. And the rc folder is something all module should use, as `VOTCARC.bash` (from `tools`) sources all files in there automatically (see https://github.com/votca/tools/blob/master/scripts/VOTCARC.bash.in#L30)

Also I thought the `xml` one should be setup as `xml/<module>` instead of `<module>/xml`, same for templates.

Anyhow, I don't have a strong opinion, so we can move it around.

@JoshuaSBrown any opinion?

**Comment by JensWehner** [comment]
true, I do not have a strong opinion but it should be consistent.
